master/buildbot/changes/base.py

```diff
@@ -52,10 +52,13 @@ def _unclaimService(self):
 
 class PollingChangeSource(ChangeSource):
 
-    def __init__(self, name=None, pollInterval=60 * 10, pollAtLaunch=False):
-        ChangeSource.__init__(self, name=name)
-        self.pollInterval = pollInterval
-        self.pollAtLaunch = pollAtLaunch
+    def __init__(self, *args, **kwargs):
+        # Provide backwards compatibility
```

Comment:
From what I know/tested I need the changes so args/kwargs are propagated to reconfigService correctly but we still have compatibility with other currently implemented Polling Sources.